openshift/templates/api/build.yaml

```diff
@@ -106,10 +106,11 @@ objects:
         role: ${ROLE_NAME}
         env: ${ENV_NAME}
     spec:
+      successfulBuildsHistoryLimit: 2
```

Comment:
is this something we might want to think about adding to all our builds?  Maybe 2-3 might be a good number.
I'm guessing you got tired of looking through the long list of builds.

Reply:
haha yeah i did get tired of sifting, so i did indeed add it to them all.